terraform/deployments/search-api-v2/datastore_schema.tf

```diff
@@ -0,0 +1,203 @@
+resource "restapi_object" "google_discovery_engine_datastore_schema" {
+  path      = "/dataStores/${google_discovery_engine_data_store.govuk_content.data_store_id}/schemas"
+  object_id = "default_schema"
+
+  data = jsonencode({
+    structSchema = {
+      "$schema" = "https://json-schema.org/draft/2020-12/schema"
+      type      = "object"
+      properties = {
+        # Unique content ID from the Publishing API (for debugging/retrieving single objects)
+        content_id = {
+          type        = "string"
+          retrievable = true
+        }
+        # Unique identifier for search quality evaluation (identical to content_id)
+        #
+        # Note: This is only used to compare results for evaluation purposes, and is not a URL
+        # despite the name. We decided against using the alternative 'uri' field name to avoid
+        # confusion with our existing 'url' field and 'uri' key property (see
+        # https://cloud.google.com/generative-ai-app-builder/docs/evaluate-search-quality)
+        cdoc_url = {
+          type        = "string"
+          retrievable = true
+        }
+        # The main title (shown in search results)
+        title = {
+          type               = "string"
+          keyPropertyMapping = "title"
+          retrievable        = true
+        }
+        # A short description (shown in search results)
+        description = {
+          type               = "string"
+          keyPropertyMapping = "description"
+          retrievable        = true
+        }
+        # Additional textual content such as keywords that should be searchable but don't form part
+        # of the main body of content
+        #
+        # Note: We've decided not to use this as structured fields don't contribute enough to
+        # relevance. Vertex does not support removing a field from the schema (as of Jan 2024) so it
+        # stays here.
+        additional_searchable_text = {
+          type       = "string"
+          searchable = true
+        }
+        # URI reference either as a relative path for GOV.UK content, or as an absolute URL for
+        # external content (used to link to the content from search results)
+        link = {
+          type        = "string"
+          retrievable = true
```

Comment:
`links` also had a `format` field. I assume this is one of ones not support by VAIS?